src/pyEQL/solution.py

```diff
@@ -3224,3 +3226,89 @@ def get_salt_list(self):  # pragma: no cover
         :py:meth:`get_salt_dict`
         """
         return self.get_salt_dict()
+
+    @classmethod
+    def from_preset(cls, preset: str) -> Solution:
```

Comment:
If you want to be extra clear about typing, you could replace `str` with `Literal['rainwater', 'seawater' ...]` to specify exactly which values are allowed